administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-04-22.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "#__extensions" ADD COLUMN "extension" VARCHAR(127) NOT NULL DEFAULT '' AFTER "template_id";
```

Comment:
```suggestion
ALTER TABLE "#__extensions" ADD COLUMN "extension" VARCHAR(127) NOT NULL DEFAULT '';
```
PostgreSQL doesn't support "BEFORE" or "AFTER" in "ALTER TABLE ... ADD COLUMN" statements.

Reply:
As agreed on Glip I will commit my suggestions. Tuan has seen them.